.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!-- CLICK "Preview" FOR INSTRUCTIONS IN A MORE READABLE FORMAT -->
```

Comment:
The rest of the files has a definition at the top between `---` markers. Is there any particular reason why this one does not have it?

Reply:
It was to specify what shows when you open an issue and have to pick between the two templates, but I'll add it to this file too in case in the future we have more pull request templates.